### DISCUSSION TITLE
Long slugs going to 404

### DISCUSSION THREAD

**Discussion by mrmathewc** [statamic/cms#4371] [Troubleshooting] [answered]
#### Issue description
I'm creating blog posts using collections and have added one of the posts that I have hosted on my WordPres website. I noticed that when the slug length was long, the route to the post doesn't respond and instead goes to a 404, however when I create a post with a relatively small slug, it loads the route/page fine.

#### Steps to reproduce the issue

1.  Create 'Blog' as collection
2. Set route to: `route: '{parent_uri}/{slug}'`
3. Create long blog post with auto-generated slug called: _"Big massive title to see if the routing works on the site"_
4. Create small blog post with auto-generated slug called: _"Test blog"_
5. Try and preview both posts, 1 works whereas the other does not (for me, anyway)

#### What's the expected result?
Expected result is long-slugs should display the route the same as smaller slugs.

#### What's the actual result?
Longer slugs don't route to template or page and instead show a 404 page.

**Comment by edalzell** [comment, marked as answer]
Sounds like a  bug to me

Opened one for you at https://github.com/statamic/cms/issues/4398
